tests/common/gu_utils.py

```diff
@@ -477,3 +480,53 @@ def expect_acl_rule_removed(duthost, rulename, setup):
         removed = len(output) == 0
 
         pytest_assert(removed, "'{}' showed a rule, this following rule should have been removed".format(cmds))
+
+
+def save_backup_test_config(duthost, file_postfix="bkp"):
```

Comment:
>save_backup_test_config [](http://example.com/codeflow?start=4&length=23)

We do have checkpoint and rollback cli.

Reply:
This code was already available as part of setup fixtures in the GCU test suites.
The main change introduced by this PR is moving the code into a separate function to avoid code duplication, as it is used by multiple setup module fixtures. Additionally, support has been added to save backup files for different namespaces on multi-ASIC platforms.
Most GCU setup module fixtures create a checkpoint and save backup configuration JSON files before the test (this block of code), then roll back to the checkpoint afterward. If the rollback fails, these backup files are used to reload the configdb.

Reply:
I understand the code could be existing in old implementation, but my point is that we should leverage the checkpoint and rollback feature.